templates/app/aspnet-core/src/MyCompanyName.MyProjectName.Domain/Data/MyProjectNameDbMigrationService.cs

```diff
@@ -89,5 +107,81 @@ private async Task SeedDataAsync(Tenant tenant = null)
 
             await _dataSeeder.SeedAsync(tenant?.Id);
         }
+
+        private async Task<bool> CheckMigrationsAsync()
+        {
+            var dbMigrationsProjectFolder = GetDbMigrationsProjectFolderPath();
+
+            if (!Directory.Exists(Path.Combine(dbMigrationsProjectFolder, "migrations")))
+            {
+                return false;
+            }
+
+            return true;
+        }
+
+        private async Task AddInitialMigration()
+        {
+            Logger.LogInformation("Creating initial migration...");
+
+            string argumentPrefix;
+            string fileName;
+
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX) || RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
+            {
+                argumentPrefix = "-c";
+                fileName = "/bin/bash";
+            }
+            else
+            {
+                argumentPrefix = "/C";
+                fileName = "cmd.exe";
+            }
+
+            var procStartInfo = new ProcessStartInfo( fileName,
+                $"{argumentPrefix} \"abp create-migration-and-run-migrator {GetDbMigrationsProjectFolderPath()}\""
+            );
+
+            try
+            {
+                Process.Start(procStartInfo);
```

Comment:
you can use this [https://github.com/abpframework/abp/blob/dev/framework/src/Volo.Abp.Cli.Core/Volo/Abp/Cli/Utils/CmdHelper.cs#L16](https://github.com/abpframework/abp/blob/dev/framework/src/Volo.Abp.Cli.Core/Volo/Abp/Cli/Utils/CmdHelper.cs#L16) .. Same things are being done in this class

Reply:
Template domain project doesn't reference Volo.Abp.Cli